tests/hook_main.py

```diff
@@ -3826,6 +3826,16 @@ def delete_order(transaction):
     transaction['skip'] = True
 
 
+@hooks.before("Orders > Charge > Charge for an Order")
+def orders_charge(transaction):
+    """
+    GET /orders/1/charge
+    :param transaction:
+    :return:
+    """
+    transaction['skip'] = True
```

Comment:
Why skipping the transaction?

Reply:
How will i get the actual stripe token for testing ?